dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -180,7 +177,24 @@ public GHContent tryRetrievingContent(GHRepository repo, String path, String bra
                 Thread.sleep(TimeUnit.MINUTES.toMillis(1));
             }
         }
-
         return listOfRepos;
     }
+
+    /**
+     * Returns a <code>java.util.Map</code> of GitHub repositories owned by a user. Returned map contains repository names
+     * mapped to their corresponding GitHub repository objects.
+     * @param user
+     * @return
+     */
+    public Map<String, GHRepository> getReposForUserAtCurrentInstant(GHMyself user) {
+        Map<String, GHRepository> repoByName = new HashMap<>();
+        if (user == null) {
+            return repoByName;
```

Comment:
Might be useful to cover this with a unit test. Was this situation covered by a test for `getGHRepositories()`?

Reply:
Added unit test for the `null` case.